pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,70 +1,131 @@
+
+## Start your full node
+Configuring and starting your full node for the first time allows it to sync with the dYdX chain network. If you saved a dYdX snapshot to your data directory, starting your node will first use that snapshot to quickly recreate most of the chain's history.
+
+1. Configure parameters in your command line. Use the following syntax:
+
+   ```bash
+   nohup dydxprotocold start
+   --p2p.seed=$SEED_LIST
+   --home=$DYDX_HOME
+   --non-validating-full-node=true
+   > /tmp/fullnode.log 2>&1 &
+   
+   # Example values
+   SEED_LIST="123@seeds.polkachu.com:123,123@seeds.kingnodes.net:123"
+   DYDX_HOME=/path/to/your/data/directory
+   ```
+
+   Using the `--p2p.seeds` command line flag, provide a comma-separated list of node URIs in the blockchain network that you are connecting to. For a list of node URIs, see the Resources page section for [Seed Nodes](../infrastructure_providers-network/resources.mdx#seed-nodes).
+   
+   Using the `--home` command line flag, provide the path to your data directory.
+
+   Once you have entered your parameters, your command should resemble the following:
+   ```bash
+   nohup dydxprotocold start --p2p.seeds="ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0@seeds.polkachu.com:23856,65b740ee326c9260c30af1f044e9cda63c73f7c1@seeds.kingnodes.net:23856,f04a77b92d0d86725cdb2d6b7a7eb0eda8c27089@dydx-mainnet-seed.bwarelabs.com:36656,20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856,c2c2fcb5e6e4755e06b83b499aff93e97282f8e8@tenderseed.ccvalidators.com:26401,4f20c3e303c9515051b6276aeb89c0b88ee79f8f@seed.dydx.cros-nest.com:26656,a9cae4047d5c34772442322b10ef5600d8e54900@dydx-mainnet-seednode.allthatnode.com:26656,802607c6db8148b0c68c8a9ec1a86fd3ba606af6@64.227.38.88:26656,4c30c8a95e26b07b249813b677caab28bf0c54eb@rpc.dydx.nodestake.top:666,ebc272824924ea1a27ea3183dd0b9ba713494f83@dydx-mainnet-seed.autostake.com:27366" --home=$DYDX_HOME --non-validating-full-node=true > /tmp/fullnode.log 2>&1 &
+   ```
+   
+   Run your command to start the full node.
+
+2. Monitor your full node's progress by tailing the log (optional).
+   
+   To tail the log, run the following command: 
+   ```bash
+   tail -f /tmp/fullnode.log
+   ```
+3. Confirm that your full node has finished syncing by comparing its current block to the dYdX chain.
+   
+   The full node is caught up with the dYdX chain head when it reaches the dYdX chain's current block.
+
+   To determine your full node's current block, use a block explorer like this example on [mintscan.io](https://www.mintscan.io/dydx).
+
+   To determine the dYdX chain's current block, use the following program: https://github.com/chiwalfrm/dydxexamples/blob/1d46b7a75499205d9c1c1986ae4ae8f21b6c1385/v4block_subscribe.py
+
+   Run the program with your full node IP address and port `26657`:
+   ```bash
+   python3 v4block_subscribe.py ws://$FULL_NODE_IP_ADDRESS:26657
+
+   # Example values
+   FULL_NODE_IP_ADDRESS=192.168.0.150
```

Comment:
where do you find this info?

Reply:
One way on Mac is running`ipconfig getifaddr en0` (wifi) or `ipconfig getifaddr en1` (ethernet)

I'm tempting to say this should be out of scope, too. There are a lot of ways to get your IP and a few factors for whether a command works or not, but at the end of the day most developers will have a way to get the one they want to use (probably by googling it), whether it's for a VM or their physical machine